problems/4.arrays.js

```diff
@@ -3,25 +3,31 @@
 * @returns {number} - The average of the first and last values
 */
 function firstAndLastAverage() {
-
+// const firstAndLastAverage =  => {
 }
 
+
+
 /** Duplicates all values inside an array.
 * Sample input: [1,4,2,5]
 * Sample output: [1,4,2,5,1,4,2,5]
 * @param {number[]} arr - The input array
 * @returns {number[]} - The duplicated array
 */
-function duplicateArray() {
-
-}
-
+function duplicateArray(arr) {
+//  let arr = arr1.slice()
```

Comment:
Try looking into concat on MDN, or looping through arr twice and pushing elements into a result array.